ymcatwincities/openy
# Openy PEF GroupEx PRO Syncer. Location filter. Substitute trainer support. Cancellation support. YGTC Backport.

Could be reviewed

Steps for review
- [ ] Reinstall the build or local dev with `openy_pef_gxp_sync` branch
- [ ] Enable ```drush en openy_mappings_feature openy_pef_gxp_sync```
- [ ] Create Program Subcategory node
![image](https://user-images.githubusercontent.com/563412/51747388-044e0400-20b2-11e9-8b6f-87897b2aad4c.png)
- [ ] Reference it from ```/admin/openy/integrations/groupex-pro/gxp```
![image](https://user-images.githubusercontent.com/563412/51747417-1fb90f00-20b2-11e9-90e3-1d5b72d7aa81.png)
- [ ] Create 2 Mapping entities ```/admin/openy/settings/mappings/mapping/add```
![image](https://user-images.githubusercontent.com/563412/51747684-f2209580-20b2-11e9-8d4d-9ef1627fbdca.png)
![image](https://user-images.githubusercontent.com/563412/51747693-fa78d080-20b2-11e9-8b2d-007f959ab77d.png)
- Remove all Class, Session, Activity nodes
- Enable Syncer ```/admin/config/system/ymca-sync```
![image](https://user-images.githubusercontent.com/563412/51747854-4fb4e200-20b3-11e9-8dc8-ff9b4c346c49.png)
- run syncer ```drush openy-pef-gxp-sync -dvy```
- Visit Group Classes page
![image](https://user-images.githubusercontent.com/563412/51747964-a3bfc680-20b3-11e9-9d5d-f0d0eeca2f74.png)
- Check there is a new feature - limit by location 
![image](https://user-images.githubusercontent.com/563412/51759015-e394a700-20cf-11e9-89eb-4456c252a41c.png)
When it's enabled - location filter hidden as well as location column in table view
